css/style.css

```diff
@@ -16,4 +29,306 @@ body {
  * - When using Flexbox, remember the items you want to move around need to be inside a parent container set to 'display: flex'
  */
 
+/* General styles, */
+h1,
+h2,
+h3 {
+  font-weight: 300;
+  line-height: 1.2em;
+  margin: 0;
+}
+
+h1 {
+  font-size: 72px;
+  color: var(--white);
+}
+h2 {
+  font-size: 60px;
+  color: var(--dark-gray);
+}
+h3 {
+  font-size: 40px;
+  color: var(--h3-color);
+}
+
+p {
+  font-size: 25px;
+}
+
+li {
+  list-style: none;
+}
+
+a {
+  color: var(--orange);
+  transition: all 0.2s ease-in;
+}
+
+hr {
+  width: 80%;
+  margin-top: 0;
+}
+
+.primary-btn {
+  color: var(--white);
+  background-color: var(--orange);
+  border: none;
+  border-radius: 0.3rem;
+  padding: 1rem 2rem;
+  font-size: 1.2rem;
+  transition: all 0.2s ease-in;
+  text-decoration: none;
+}
+
+.primary-btn:hover {
+  color: var(--orange);
+  background-color: var(--white);
```

Comment:
I believe the best option here, in the case of keeping the white background with orange text, is to add an outline to make the button's boundaries visible. As it is, the button blends in with the background of the page, and only the text is visible.